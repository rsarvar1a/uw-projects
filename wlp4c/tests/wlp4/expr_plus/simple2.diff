2c2
< 25
---
> 0
