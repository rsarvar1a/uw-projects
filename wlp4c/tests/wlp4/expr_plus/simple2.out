Enter value for register 1: Enter value for register 2: Running MIPS program.
25
MIPS program completed normally.
$01 = 0x00000006   $02 = 0xfffffffe   $03 = 0x0000000a   $04 = 0x00000004   
$05 = 0x0000000c   $06 = 0x00000cac   $07 = 0x00000006   $08 = 0x00000008   
$09 = 0x00000000   $10 = 0x00000cb4   $11 = 0x00000001   $12 = 0x00000000   
$13 = 0x00000000   $14 = 0x00000000   $15 = 0x00000000   $16 = 0x00000000   
$17 = 0x00000000   $18 = 0x00000000   $19 = 0x00000000   $20 = 0x00000000   
$21 = 0x00000000   $22 = 0x00000000   $23 = 0x00000000   $24 = 0x00000000   
$25 = 0x00000000   $26 = 0x0000031c   $27 = 0x000004f4   $28 = 0x00000184   
$29 = 0x01000000   $30 = 0x01000000   $31 = 0x8123456c   
